tests.py

```diff
@@ -261,3 +272,49 @@ def test_stream_request_with_dict_mock(self):
     def test_non_stream_request(self):
         r = requests.get('http://domain.com/')
         self.assertEqual(r.raw.read(), b'')
+
+
+class RememberCalledTest(unittest.TestCase):
+
+    def several_calls(self, count, method, *args, **kwargs):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/martins-laucis/httmock/file/2632856329/issues/source?bid=3318514&fileBranchId=3318515#l279)